src/sdk/staking.ts

```diff
@@ -267,53 +271,71 @@ async function metaMyStake(web3: Web3, address: string, account: string, isV2: b
  *  @param {string} account account details to fetch.
  * @returns {Promise<Stake | null>}
  */
-async function metaMyGovStake(web3: Web3, account: string): Promise<MyStake | null> {
+async function metaMyGovStake(web3: Web3, account: string): Promise<MyStake[] | null> {
```

Comment:
instead of rewriting the code here modify this method to receive an optional "address" of the gov staking contract.
then from the calling method call this twice, for example
```
async function getGovStakes(web3,account) {
 return Promise.all([metaMyGovStake[web3,account,v1Address],metaMyGovStake(web3,account,v2Address)])
}
```

Reply:
I went with what we used for the simplestaking contracts with .map
there is not really a benefit for having a separate function I thought?